src/scene/comparisons.ts

```diff
@@ -75,6 +75,7 @@ export const getElementContainingPosition = (
   elements: readonly ExcalidrawElement[],
   x: number,
   y: number,
+  excludedType?: ExcalidrawElement["type"],
```

Comment:
since this param is only used in one place where perf isn't an issue, I'd just filter the elements upstream. Adding it as a param inside her isn't semantic and really ad-hoc.